ga4gh/backend.py

```diff
@@ -708,3 +729,90 @@ def __init__(self, dataDir):
             dataset = datasets.FileSystemDataset(datasetDir)
             self._datasetIdMap[dataset.getId()] = dataset
         self._datasetIds = sorted(self._datasetIdMap.keys())
+
+
+class GeneotypePhenotypeBackend(AbstractBackend):
```

Comment:
I don't think we should view this as a seperate backend, but rather as a part of the data model. See my earier comments about AssociationSets, etc.